### PR TITLE
Anahid enhancements

### PR DESCRIPTION
**Pull request by nmellado**
List of changes:
* Various improvements coming form @Ana873331 and @CelMichaud work + fixes from @hoshiryu (camera auto setting),
* new cmake option `RADIUM_FAST_MATH`, to control the fastmath optimizations (when supported by compiler). As this optimization makes some tests failing, it is set to `FALSE` by default. 
* new implementation for check_NaN in the Animation package, using Eigen power (fixes broken tests in release mode)
* Add new convenience function `RadiumEngine::getMesh` to ease access to Meshes in plugins 176b9f3
* Cleaning in `ConfigureCompiler.cmake`, including a fix when no compile mode is set (see 4e0fe1a)

### CODE REVIEW

**Comment by nmellado** [comment]
This PR is now ready for reviews.
See main description for list of changes

File: src/Core/Mesh/DCEL/Iterator/Edge/EFIterator.inl [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            f2 = h2->F();
+            if (f2 == f1) break;
+            L.push_back(f2);
+            h2 = h2->Next()->Twin();
+        }
+    }
+
+    return L;
+}
+*/
```
Comment (louen) [inline src/Core/Mesh/DCEL/Iterator/Edge/EFIterator.inl]: Is this code dead ?

Comment (nmellado) [reply src/Core/Mesh/DCEL/Iterator/Edge/EFIterator.inl]: Files have been removed. Please ignore

**Comment by nmellado** [comment]
PR ready for review again.

File: src/Core/Mesh/DCEL/Iterator/Edge/EdgeIterator.inl [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+namespace Ra {
+namespace Core {
+
+/// CONSTRUCTOR
+template < typename OBJECT >
+EIterator< OBJECT >::EIterator( HalfEdge_ptr& he ) :
+    Iterator< OBJECT >(){
+    this->m_he = he;
+}
```
Comment (louen) [inline src/Core/Mesh/DCEL/Iterator/Edge/EdgeIterator.inl]: Why so much whitespace ? We're not paid by line of code, aren't we ?

**Pull request merged**